frontend/src/components/SideNav.tsx

```diff
@@ -448,7 +448,10 @@ export class SideNav extends React.Component<SideNavInternalProps, SideNavState>
           )}
           <hr className={classes(css.separator, collapsed && css.collapsedSeparator)} />
           <div
-            className={classes(css.indicator, page !== RoutePage.ARCHIVE && css.indicatorHidden)}
+            className={classes(
+              css.indicator,
+              page !== RoutePage.ARCHIVED_RUNS && css.indicatorHidden,
```

Comment:
nit: change to `![RoutePage.ARCHIVED_RUNS, RoutePage.ARCHIVED_EXPERIMENTS].includes(page) && css.indicatorHidden`?

Reply:
done